Resources/Prototypes/DeltaV/Voice/speech_emote_sounds.yml

```diff
@@ -28,25 +28,25 @@
       collection: HarpyGrowls
     Purr:
       collection: HarpyPurrs
-    Ring:
+    HarpyRing:
       collection: HarpyRings
-    HonkHarpy:
+    HarpyHonk:
       collection: HarpyHonks
-    Pew:
+    HarpyPew:
       collection: HarpyPews
-    Bang:
+    HarpyBang:
       collection: HarpyBangs
-    Beep:
+    HarpyBeep:
       collection: HarpyBeeps
-    Rev:
+    HarpyRev:
       collection: HarpyRevs
-#    Click:
-#      collection: HarpyClicks
+    Click:
```

Comment:
Why are some HarpySound: collection and others are just Sound: collection?

Reply:
Because HarpySounds collections are vastly larger than existing sound collections, and to futureproof it against people messing with existing ones, I simply migrated the names of the collections to ones for the Species. So that if someone for example makes a Clicks sound collection, it won't conflict with HarpyClicks(which contains a shitload of click sounds)

Reply:
ahh okay, makes sense.